contrib/scripts/export-creds.sh

```diff
@@ -0,0 +1,106 @@
+#!/usr/bin/env sh
+
+set -euo pipefail
+
+umask 077
+
+err_report() {
+    echo "[ERROR] Command on line $1 exited with exit code $2"
+    exit $2
+}
+trap 'err_report $LINENO $?' ERR
```

Comment:
SC3047: In POSIX sh, trapping ERR is undefined.

Reply:
wonder if there's an alternative here, can't stand when shell scripts silently fail.